provider/ecs/ecs.go

```diff
@@ -178,17 +178,34 @@ func wrapAws(ctx context.Context, req *request.Request) error {
 	return req.Send()
 }
 
-func (p *Provider) loadECSConfig(ctx context.Context, client *awsClient) (*types.Configuration, error) {
-	var ecsFuncMap = template.FuncMap{
-		"filterFrontends":         p.filterFrontends,
-		"getFrontendRule":         p.getFrontendRule,
-		"getBasicAuth":            p.getBasicAuth,
-		"getLoadBalancerMethod":   p.getLoadBalancerMethod,
-		"getLoadBalancerSticky":   p.getLoadBalancerSticky,
-		"hasStickinessLabel":      p.hasStickinessLabel,
-		"getStickinessCookieName": p.getStickinessCookieName,
+// generateECSConfig fills the config template with the given instances
+func (p *Provider) generateECSConfig(services map[string][]ecsInstance, ecsFuncMap template.FuncMap) (*types.Configuration, error) {
```

Comment:
Could you explains `ecsFuncMap`? It's always `nil`...

Reply:
For ease of testing. I wanted to be able to have a failure case, but I see that is useless now. I will change.